src/novmm/loader/linux_setup.go

```diff
@@ -21,9 +21,11 @@ package loader
 #include <string.h>
 
 #ifdef __x86_64__
-#include <x86_64-linux-gnu/asm/bootparam.h>
+//#include <x86_64-linux-gnu/asm/bootparam.h>
+#include <asm/bootparam.h>
 #else
-#include <i386-linux-gnu/asm/bootparam.h>
+//#include <i386-linux-gnu/asm/bootparam.h>
+#include <asm/bootparam.h>
```

Comment:
The #ifdef no longer makes sense and you can just unconditionally ``#include asm/bootparam.h``.